mods/d2k/maps/atreides-01a/map.yaml

```diff
@@ -145,8 +147,8 @@ Rules:
 			Minimum: 1
 			Maximum: 1
 	construction_yard:
-		Production:
-			Produces: Building
+		Buildable:
+			Prerequisites: ~disabled
```

Comment:
Shouldn't `construction_yard` be unabuildable by default?
If so, please remove this (also from the other missions).